src/backend/web/handlers/webhooks.py

```diff
@@ -0,0 +1,140 @@
+import uuid
+
+from flask import Blueprint, redirect, request, url_for
+from google.appengine.ext import ndb
+from pyre_extensions import none_throws
+
+from backend.common.auth import current_user
+from backend.common.consts.client_type import ClientType
+from backend.common.helpers.tbans_helper import TBANSHelper
+from backend.common.models.account import Account
+from backend.common.models.mobile_client import MobileClient
+from backend.web.decorators import require_login
+from backend.web.profiled_render import render_template
+
+blueprint = Blueprint("webhooks", __name__, url_prefix="/webhooks")
+
+
+@blueprint.route("/add", methods=["GET"])
+@require_login
+def webhook_add():
+    template_values = {
+        "error": request.args.get("error"),
```

Comment:
We migrated to using the session for this sort of stuff as opposed to passing it through the URL args. See https://github.com/the-blue-alliance/the-blue-alliance/wiki/Web#passing-data-between-pages

Reply:
updated, thanks for the tip!